### PR TITLE
objdump intel syntax

### PR DESCRIPTION
**Pull request by ttomsa**
more readable and the first operand is the destination like arm64/ptx

### CODE REVIEW

**Comment by geohot** [comment]
Needs a test

File: test/test_objdump.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from tinygrad import Device
+from tinygrad.helpers import cpu_objdump
```
Comment (geohot) [inline test/test_objdump.py:2]: Should be a `disassemble` method on the Compiler, see CUDA/AMD/METAL/QCOM

Comment (ttomsa) [reply test/test_objdump.py]: This is a bit outside the intended scope of this pr. Regardless I decided to just use llvm-objdump as it works on windows and macos.

**Comment by geohot** [comment]
I shouldn't have merged this with the try except. I don't have llvm-objdump on my computer.

**Comment by geohot** [comment]
That test also passes if you don't have llvm-objdump.

**Comment by ttomsa** [comment]
I didn't change the test cause the CI has llvm-objdump. Llvm-objdump comes with llvm not clang. I think it's better cause it works on other platforms. If you're on windows you just need llvm whereas with objdump you need wsl/mingw.

**Pull request merged**